### ISSUE TITLE
Detect course does not have files enabled

### ISSUE THREAD

**Issue by rutgervana** [BoYanZh/Canvas-Syncer#18]
Some courses disable the canvas files page momentarily. This will break the syncing of files. 
Giving the following debug trace:
```
Unexpected Error: TypeError. Please check your network and your token!
Traceback (most recent call last):
  File "/home/rutger/.local/lib/python3.7/site-packages/canvassyncer/__main__.py", line 496, in run
    Syncer.sync()
  File "/home/rutger/.local/lib/python3.7/site-packages/canvassyncer/__main__.py", line 394, in sync
    self.checkNewFiles()
  File "/home/rutger/.local/lib/python3.7/site-packages/canvassyncer/__main__.py", line 336, in checkNewFiles
    for info in self.getCourseTaskInfo(courseID):
  File "/home/rutger/.local/lib/python3.7/site-packages/canvassyncer/__main__.py", line 285, in getCourseTaskInfo
    folders, files = self.getCourseFiles(courseID)
  File "/home/rutger/.local/lib/python3.7/site-packages/canvassyncer/__main__.py", line 242, in getCourseFiles
    if f['folder_id'] not in folders.keys(): continue
TypeError: string indices must be integers
```
Is there a way to handle this e.g check whether the course has a files page? Maybe also log it as a separate  error, the current one does not really give a hint to what is going wrong. The short term fix would be to remove the course and at it back when the files page is enabled again, but this is not ideal.

**Comment by BoYanZh** [comment]
hmm, I have not met such a situation. Do you have any clue about how to judge whether the files page is enabled?

**Comment by rutgervana** [comment]
I did some debugging and instead of a file in the `getCourseFiles` I get an error message.
This is the print of the `files` variable where the function errors.
```
{'status': 'unauthorised', 'errors': [{'message': 'user not authorised to perform that action'}]}
``` 
So maybe a check whether status is unauthorized?

**Comment by BoYanZh** [comment]
Oh, I see. It seems that it has been fixed on #17 but I forget to update it on pypi. Now it should be available. <https://pypi.org/project/canvassyncer/1.2.11/>
